### PR TITLE
fixed links and added missing content

### PR DESCRIPTION
**Pull request by wolfdancer**
Fixed the links to API doc.  Added missing content for agent checks and hostinfo checks.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-415a2008cd/how-to/)

File: content/rackspace-monitoring/rackspace-monitoring-checks-and-alarms.md [hunk 1]
```diff
@@ -16,28 +16,20 @@ Because a check does not trigger any alerts by itself, collecting more data is a
 
 Rackspace Monitoring uses *alarms* to analyze the data that is collected by a check. The alarm criteria contains the logic to process this data and convert the alarm into one of the three states: OK, WARNING, and CRITICAL.
 
-### Where to configure checks and alarms
+## Where to configure checks and alarms
```
Comment (DonNateR) [inline content/rackspace-monitoring/rackspace-monitoring-checks-and-alarms.md]: @wolfdancer: Could you change all H2 level headings back to H3? Our style guidelines state that we only use H3, H4, and H5 level headings.

**Comment by wolfdancer** [comment]
@DonNateR Done.  Sorry I missed that.  Thanks!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b154535969/how-to/)

**Pull request merged**